controller/AuthoringTool.php

```diff
@@ -106,18 +104,6 @@ public function launch(): void
         $this->forward('run', null, null, $_GET);
     }
 
-    /**
-     * @throws ContainerExceptionInterface
-     * @throws NotFoundExceptionInterface
-     */
-    private function isFeatureTaoAsToolEnabled(): bool
-    {
-        return $this->getServiceManager()
-            ->getContainer()
-            ->get(FeatureFlagChecker::class)
-            ->isEnabled(FeatureFlagCheckerInterface::FEATURE_FLAG_TAO_AS_A_TOOL);
-    }
-
```

Comment:
Isn't this part of ticket: [AUT-3576](https://oat-sa.atlassian.net/browse/AUT-3576)

Reply:
no, this FF is not mentioned on that ticket